apps/ledger-live-mobile/src/screens/Manager/AppsScreen.tsx

```diff
@@ -304,7 +307,7 @@ const AppsScreen = ({
     ),
     [
       appFilter,
-      blockNavigation,
```

Comment:
Where did this go?

Reply:
I see, I liked the generic wording in case we introduced another case for the block to happen, but 🤷🏼‍♂️ I guess it's ok.

Reply:
I changed the name to "pendingInstalls" specifically because I'm using the fact that there are apps installing to block the changing of languages. It's specifically to prevent us from adding another case to this and end up messing with this block.

Reply:
👍🏼